spec.html

```diff
@@ -209,6 +209,9 @@ <h1>primitive value</h1>
       <emu-note>
         <p>A primitive value is a datum that is represented directly at the lowest level of the language implementation.</p>
       </emu-note>
+      <emu-note>
+        <p>The value of Null type has additional usage besides being one of primitive values: <emu-xref href="#sec-ecmascript-language-types-null-type"></emu-xref></p>
```

Comment:
"Being a primitive value" isn't a usage, it's what it is, so this link isn't an additional use of that.

Reply:
I thought about it in first. But then thought about a chain between 1.2.3. and leaved it untouched.
I need more understanding, there must be the other way to explain.